wordpress-mobile/WordPress-Android
# Adjust comment Like icon boundary to correct rotation animation

Fixes #6101.

Simple fix, the icon's ImageView had unequal padding so the animation looked a bit wonky.

To test:
* Head to Comments screen and select a comment
* Like/Unlike and make sure rotation animation is centered
